README.md

```diff
@@ -71,113 +73,147 @@ To compile and build the `broadinstitute/firecloud-ui` docker image, run
+and watch them execute in a browser. _IMPORTANT_: Be careful about interacting with your computer
+while running tests this way because stray keyboard and mouse interactions can interfere with test
+execution. It is recommended to use this only when writing/debugging specific tests, not running the
+entire test suite. 
 
 #### Set Up
 
-First run render script to generate necessary configs:
+Before you run the tests for the first time, make sure you have chromedriver installed. On Mac OS,
+Homebrew (`brew`) is recommended:
 
 ```bash
-cd automation
-./render-local-env.sh $PWD $(cat ~/.vault-token) <"qa" or "dev">
+brew install chromedriver
 ```
 
+If your version of Chrome is 61 or later, check `chromedriver --version` to make sure you're up to
+at least 2.32. If not, use `brew upgrade chromedriver` to update.
+ 
+Also run the config render script. This will render the necessary `application.conf` and
+`firecloud-account.pem` for the tests. From the `automation` directory:
 
-If you have a local UI running, you will need to go into `automation/src/test/resources` and edit the `baseURL` in `application.conf`:
-
-```
-baseUrl = "http://local.broadinstitute.org/"
+```bash
+./render-local-env.sh $PWD $(cat ~/.vault-token) <dev or qa>
 ```
 
+#### Running tests
 
-When starting your UI, run:
+##### From IntelliJ
 
-```bash
-FIAB=true ENV=qa ./config/docker-rsync-local-ui.sh
+First, you need to set some default VM parameters for ScalaTest run configurations. In IntelliJ, go
+to `Run` > `Edit Configurations...`, select `ScalaTest` under `Defaults`, and add these VM
+parameters:
+```
+-Djsse.enableSNIExtension=false -Dheadless=false
 ```
 
+Also make sure that there is a `Build` task configured to run before launch.
 
-#### Running tests
-
-##### From intellij
-
-Edit your run config defaults for ScalaTest.
-Add this to the VM parameters: `-Djsse.enableSNIExtension=false  -Dheadless=false`
+Now, simply open the test spec, right-click on the class name or a specific test string, and select
+`Run` or `Debug` as needed. A good one to start with is `GoogleSpec` to make sure your base
+configuration is correct.
 
 ##### From the command line
 
 To run all tests:
 
 ```bash
 sbt test -Djsse.enableSNIExtension=false -Dheadless=false
-sbt clean
 ```
 
 To run a single suite:
 
 ```bash
 sbt -Djsse.enableSNIExtension=false -Dheadless=false "testOnly *GoogleSpec"
-sbt clean
 ```
 
 To run a single test within a suite:
 
 ```bash
 # matches test via substring
 sbt -Djsse.enableSNIExtension=false -Dheadless=false "testOnly *GoogleSpec -- -z \"have a search field\""
-sbt clean
 ```
 
 For more information see: http://www.scala-sbt.org/0.13/docs/Testing.html#Test+Framework+Arguments
 
+##### Against a local UI
+
+To run tests against firecloud-ui running on your local workstation, you will need to edit the
+`baseURL` config parameter in the `automation/src/test/resources` produced by `render-local-env.sh`:
```

Comment:
should be `src/test/resources/application.conf`, not `src/test/resources`

Reply:
Thanks, nice catch!